setup.py

```diff
@@ -64,11 +71,11 @@ def read(file_name):
     install_requires=required_packages,
 
     extras_require={
-        'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'mock', 'sagemaker', 'numpy']
+        'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'mock', 'sagemaker==1.16.2']
```

Comment:
==1.16.2, or >=1.16.2? If a breaking change is released to SageMaker Python SDK, the tests should fail.